modules/sqs/src/main/scala/vinyldns/sqs/queue/SqsMessageQueue.scala

```diff
@@ -163,74 +180,28 @@ object SqsMessageQueue {
 
     new SqsMessageQueue(queueUrl, client)
   }
-}
-
-trait SqsConversions extends ProtobufConversions {
```

Comment:
I deleted the SqsConversions, moved "message type" things to `SqsMessageType`; moved "message" things to `SqsMessage`, and moved queue things to `SqsMessageQueue`